docs/install/proxmox/proxmox-setup.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Installing Colony with Promox
+description: Follow the steps to install Colony with Proxmox
+sidebar_position: 2
+---
+
+## Installing Colony with Proxmox
+
+This guide walks through the setup of Laptop-0, a machine for running Colony, and 3 additional VM's for Colony to auto discovery and install a K3S cluster.  Some items to note:
+
+- Colony requires Docker.
+- If you have a firewall on your host machine (laptop 0) it needs to allow the following ports (ports TBA).
+
+Warning: Colony will be able to PXE boot all machines in your network.
+
+## Getting Started
+
+The steps here are optional, but we recommend completing these to create a Proxmox template VM that you can clone to for better performance and asset creation.
+
+### Debian image
+
+These steps use a `debian cloudinit image for laptop VM`.
+
+- You will need to install `libguestfs-tools` to interact with cloud-init image.
+
+    ```shell
+    apt update
+    apt install libguestfs-tools
+    ```
+
+- Commands should be run in Proxmox shell, which can be accessed through the Proxmox UI through `datacenter > <node-name> > shell`, or through ssh.
+
+- Might to prepend sudo if you running user different than `root`
+
+Download debian cloud init image [debian download image](https://cloud.debian.org/images/cloud/). Create the directory anywhere. Select the correct architecture. This example runs on x86
+
+```shell
+mkdir debian-golden-img
+
+wget https://cloud.debian.org/images/cloud/bookworm/20241004-1890/debian-12-genericcloud-amd64-20241004-1890.qcow2
+
+```
+
+### Guest agent and VIM
+
+Install `qemu-guest-agent` and `vim`. This enables information to report back from the guest machine to the host. Allows power off commands to pass to the guest.
+
+```shell
+virt-customize -a debian-12-genericcloud-amd64-20241004-1890.qcow2 --install qemu-guest-agent
+virt-customize -a debian-12-genericcloud-amd64-20241004-1890.qcow2 --install vim
+```
+
+### Helm and Kubectl
+
+Install `kubectl` and `Helm` binary. Download and copy to `/usr/local/bin`
+
+For helm
+
+```shell
+wget https://get.helm.sh/helm-v3.16.2-linux-amd64.tar.gz
+tar xvf helm-v3.16.2-linux-amd64.tar.gz
+virt-customize -a debian-12-genericcloud-amd64-20241004-1890.qcow2 --copy-in ./linux-amd64/helm:/usr/local/bin/
+
+rm -rf ./linux-amd64
+```
+
+For kubectl
+
+```shell
+# kubectl
+wget "https://dl.k8s.io/release/v1.31.2/bin/linux/amd64/kubectl"
+virt-customize -a debian-12-genericcloud-amd64-20241004-1890.qcow2 --copy-in ./kubectl:/usr/local/bin/
+```
+
+### Resize config
+
+Remove machine config and resize root disk to `32 Gb`. This allows new virtual machines created from the template to have unique machine ID.
```

Comment:
Prepare virtual machine image by increasing the disk size to `32GB` and removing machine-specific information generated in previous step. This ensure that the image can be safely cloned or reused without conflicts.